routes/mapper.py

```diff
@@ -162,25 +162,25 @@ def __init__(self, obj, resource_name=None, collection_name=None,
                 self.formatted = True
         self.add_actions(actions or [], **kwargs)
 
-    def connect(self, *args, **kwargs):
+    def connect(self, routename, path=None, **kwargs):
```

Comment:
Oh, I didn't see that the old code (Routes 2.3) allow any number of positional argument. Was it a deliberate choice?

IMHO dims's code is better (explicitly only accept two positional parameters).

Note: With Routes 2.3, is it possible to get 'path' key in kwargs?